rasa/cli/test.py

```diff
@@ -83,6 +84,12 @@ def run_core_test(args: argparse.Namespace) -> None:
     if isinstance(args.model, list) and len(args.model) == 1:
         args.model = args.model[0]
 
+    if args.model is None:
+        rasa.shared.utils.cli.print_error(
+            "No model provided. Please make sure to specify the model to test with '--model'."
+        )
+        return
+
     if isinstance(args.model, str):
         model_path = rasa.cli.utils.get_validated_path(
             args.model, "model", DEFAULT_MODELS_PATH
```

Comment:
The default value doesn't make sense then, does it?

Reply:
It does. The errors happens if `args.model = [None]`.

Reply:
but `[None]` already means that there is no model in the `DEFAULT_MODELS_PATH`, no?

Reply:
No. The default value of `--model` for `rasa test core` is set to the following: https://github.com/RasaHQ/rasa/blob/master/rasa/cli/arguments/test.py#L155
In case there is no model folder, it is set to `[None]`. As it is a list of length 1 we extract the first value with is `None`. As `None` is not of type `str` it goes into the `else` part, which failed.